components/ws-daemon/pkg/content/service.go

```diff
@@ -153,7 +153,7 @@ func (s *WorkspaceService) InitWorkspace(ctx context.Context, req *api.InitWorks
 	var (
 		wsloc string
 	)
-	if req.FullWorkspaceBackup {
+	if req.FullWorkspaceBackup || req.PersistentVolumeClaim {
 		if s.runtime == nil {
 			return nil, status.Errorf(codes.FailedPrecondition, "full workspace backup is not available - not connected to container runtime")
 		}
```

Comment:
I know this wasn't introduced in this PR, but this hasn't made sense in a while; even less so.

Neither FWB nor PVC require this connection anymore. Please remove.